src/index.html

```diff
@@ -11,8 +12,66 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <!--#google Font-->
+    <!--crossorigin-->
+    <link
+      rel="stylesheet"
+      href="./style.scss"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
```

Comment:
The 'crossorigin' attribute is missing in this preconnect link to 'https://fonts.gstatic.com'. Adding 'crossorigin' can improve performance by allowing the browser to make the connection early.